builtin/providers/azurerm/provider.go

```diff
@@ -85,7 +85,8 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_lb_probe":                resourceArmLoadBalancerProbe(),
 			"azurerm_lb_rule":                 resourceArmLoadBalancerRule(),
 
-			"azurerm_managed_disk": resourceArmManagedDisk(),
+			"azurerm_managed_disk":  resourceArmManagedDisk(),
+			"azurerm_managed_image": resourceArmManagedImage(),
```

Comment:
I went back and forth on the naming - there's a "VirtualMachineImage" that we also have now and it was a bit difficult to mentally separate the two, especially when dealing with VM config. That's why I added the "managed" prefix to my code, but I am certainly open to changing this.